store.html

```diff
@@ -0,0 +1,130 @@
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="stylesheet" href="css/store.css">
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+    <script
+      src="https://kit.fontawesome.com/a6d80728cd.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <img src="img/karma-logo.svg" alt="karma logo" />
+      <nav>
+        <a href="/index.html">Meet Karma</a>
+        <a href="">How it Works</a>
+        <a href="">Store</a>
+        <a href="">Blog</a>
+        <a href="">Help</a>
+        <a href="">Login</a>
+      </nav>
+    </header>
+    <main>
+      <section>
```

Comment:
You don't need this section wrapper. I would instead change your divs below to sections to make your HTML more semantic